rcl_interfaces/msg/ParameterType.msg

```diff
@@ -7,8 +7,9 @@ uint8 PARAMETER_BOOL=1
 uint8 PARAMETER_INTEGER=2
 uint8 PARAMETER_DOUBLE=3
 uint8 PARAMETER_STRING=4
-uint8 PARAMETER_BYTE_ARRAY=5
-uint8 PARAMETER_BOOL_ARRAY=6
-uint8 PARAMETER_INTEGER_ARRAY=7
-uint8 PARAMETER_DOUBLE_ARRAY=8
-uint8 PARAMETER_STRING_ARRAY=9
+uint8 PARAMETER_YAML=5
```

Comment:
minor issue: I would make the new value 10, in this way you have at least some backward compatibility...